README.md

```diff
@@ -44,7 +44,7 @@ You can open example projects for [JVM](example-jvm) or [JS](example-js) to get
 ```kotlin
 
 import kotlinx.serialization.*
-import kotlinx.serialization.json.JSON
+import kotlinx.serialization.json.Json
```

Comment:
`JSON` inside the code snippet is not renamed

Reply:
I also believe that there are some .md files in the `docs` folder which also need to be updated

Reply:
I think I've got them all changed now. (Except for the change log, since I assume those should stay the same as the old names)